aries_cloudagent/protocols/connections/v1_0/manager.py

```diff
@@ -283,13 +316,32 @@ async def create_request(
             A new `ConnectionRequest` message to send to the other agent
 
         """
+        # Mediation setup
+        mediation_mgr = MediationManager(self._session)
+        mediation_record = None
+        keylist_updates = None
+        if mediation_id:
+            mediation_record = await MediationRecord.retrieve_by_id(
+                self._session, mediation_id
+            )
+            if mediation_record.state != MediationRecord.STATE_GRANTED:
+                raise ConnectionManagerError(
+                    "Medation is not granted for mediation identified by "
+                    f"{mediation_record.mediation_id}"
+                )
+
+        my_info = None
         wallet = self._session.inject(BaseWallet)
+        print(connection)
```

Comment:
probably this is for debugging and it could go away?

Reply:
How embarrassing! That print statement will be removed, thanks.